Add an entry in glossary for GVR (#42184)
kubernetes/website: content/en/docs/reference/glossary/group-version-resource.md

Reviewer: At the HTTP level, `/api/v1/namespaces/default/pods/hello-world-1` is a resource. So is `/api/v1/namespaces/default/pods` (it's a collection of Pods), but I don't think it's quite what people think of when they write GVK or GVR.

I don't have a good answer here. Something to watch out for though.

---

Resources also mean the cgroup kind: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/

---

> title: Group Version Resource

Reviewer: ```suggestion
title: Group, Version, Resource
```

id: gvr
date: 2023-07-24

> full_link: /docs/concepts/architecture/unknown-version-interoperability-proxy#Interoperability-proxying

Reviewer: ```suggestion
```

short_description: >

>   The group, version and resource of a Kubernetes API resource. 

Reviewer: ```suggestion
  The API group, API version and name of a Kubernetes API. 
```

> aka:

Reviewer: ```suggestion
aka: ["GVR"]
```

tags:
- architecture
---
Means of representing unique Kubernetes API resource.

<!--more-->

> GVRs are the group, version, resource of an API resource that are used to define and distinguish different Kubernetes objects.

Reviewer: ```suggestion
Group, Version, Resources (GVRs) specify the API group, API version, and resource (name for the object kind as it appears in the URI) associated with accessing a particular id of object in Kubernetes.
GVRs let you define and distinguish different Kubernetes objects, and to specify a way of accessing
objects that is stable even as APIs change.
```

---

> title: Group, Version, Resource

Reviewer 2: Group Version Resource

Author: Done.

id: gvr
date: 2023-07-24
short_description: >
  The API group, API version and name of a Kubernetes API. 

aka: ["GVR"]
tags:
- architecture
---
Means of representing unique Kubernetes API resource.

<!--more-->

> Group, Version, Resources (GVRs) specify the API group, API version, and resource (name for the object kind as it appears in the URI) associated with accessing a particular id of object in Kubernetes.

Reviewer 2: nit: The `id` is `gvr`, should the title use `Group Version Resource`?
nit: Group Version Resource (GVRs) specify the API group, version and resource ...

Author: Done.
